defaults/main.yml

```diff
@@ -84,6 +84,24 @@ archivematica_src_ss_environment:
   SS_GUNICORN_BIND: "127.0.0.1:8001"
 
 
+#
+# Logging settings
+#
+
+archivematica_src_dashboard_log_path: "/var/log/archivematica/dashboard"
```

Comment:
This is confusing, the log path is `/var/log/archivematica/MCPCLient/MCPClient.log` , may be we can find a better name for this setting? ( `archivematica_src_dashboard_logdir`?)